src/containerapp/azext_containerapp/_params.py

```diff
@@ -177,6 +177,7 @@ def load_arguments(self, _):
         c.argument('platform_reserved_cidr', options_list=['--platform-reserved-cidr'], help='IP range in CIDR notation that can be reserved for environment infrastructure IP addresses. It must not overlap with any other Subnet IP ranges')
         c.argument('platform_reserved_dns_ip', options_list=['--platform-reserved-dns-ip'], help='An IP address from the IP range defined by Platform Reserved CIDR that will be reserved for the internal DNS server.')
         c.argument('internal_only', arg_type=get_three_state_flag(), options_list=['--internal-only'], help='Boolean indicating the environment only has an internal load balancer. These environments do not have a public static IP resource, therefore must provide infrastructureSubnetResourceId if enabling this property')
+        c.argument('infrastructure_resource_group', options_list=['--infrastructure-resource-group', '-i'], help='Name for resource group that will contain infrastructure resources. If not provided, the default resource group name will be used.')
```

Comment:
We will move all the GA commands to core in this PR: https://github.com/Azure/azure-cli/pull/27078.
For arguments which only existing in extension containerapp,  please add tag `is_preview=True` for it.

Reply:
I'll add the flag. @pagariyaalok do you agree?